### PR TITLE
Run testsuite in circleci

### PR DESCRIPTION
**Pull request by severinson**
┆Issue is synchronized with this [Jira Task](https://gr-oss.atlassian.net/browse/BATCH-102) by [Unito](https://www.unito.io)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 version: 2.1
+orbs:
```
Comment (kannon92) [inline .circleci/config.yml:2]: I thought the general consensus was the phase out CircleCI for GHA.

Comment (Kotwic4) [reply .circleci/config.yml:2]: yet armada was not moved away from CircleCI in last 4 months and this PR was fixing releasing c# client

Comment (kannon92) [reply .circleci/config.yml:2]: Yea we have been working on this over time.  Some areas we find difficult are how to address release artifacts without leaking secrets in github.  We also were given a series of tasks to research before we fully replace circle ci.  

One of the last steps is to remove the release stage from circle CI but we have a few items that are of higher priority.  

Thank you for looking into this!

**Comment by kannon92** [comment]
We removed circle ci so this PR is no longer relevant.

**Pull request closed without merging**